network/dag/dag.go

```diff
@@ -374,7 +382,9 @@ func indexClockValue(tx stoabs.WriteTx, transaction Transaction) error {
 		return err
 	}
 
-	log.Logger().Tracef("storing transaction logical clock, txRef: %s, clock: %d", ref.String(), clockKey)
+	log.Logger().
+		WithField("txRef", ref).
+		Tracef("Storing transaction logical clock (LC: %d)", clockKey)
```

Comment:
Isn't clockKey also a field?

Reply:
I decided not to, since it's probably not a field that reoccurs in many more logs in such a way that you want to trace it. In my opinion you make stuff that you want to filter/search on or trace a field.